backend/dal/Migrations/v01.07.00/Up/PostUp/01-ViewProperties.sql

```diff
@@ -0,0 +1,221 @@
+﻿PRINT 'Updating View_Properties'
```

Comment:
Can you just post your changes really quick?

Reply:
actually I can just do a diff.

Reply:
If your PR is merged after Devin's, make sure you update this with Devin's changes or you'll lose his work

Reply:
Was adding the RowVersion